modules/extensions-mule-support/src/test/resources/salutation-operations-def.xml

```diff
@@ -104,4 +98,24 @@
         </operation:body>
     </operation:def>
 
+    <operation:def name="helloPlace" displayName="Hello to a place" summary="Says Hello to a place">
+        <operation:parameters>
+            <operation:parameter name="greeting" type="string" summary="The greeting message"/>
+            <operation:optional-parameter name="city" type="string" summary="The name of a city to say hello to"/>
+            <operation:optional-parameter name="country" type="string" summary="The name of a country to say hello to"/>
+            <operation:exclusive-optionals exclusiveOptionals="city,country" oneRequired="true"/>
+        </operation:parameters>
+        <operation:output>
+            <operation:payload-type type="string"/>
+            <operation:attributes-type type="string"/>
+        </operation:output>
+        <operation:body>
+            <this:hello-world greeting="#[params.greeting]"
```

Comment:
there should be a test which calls this operation correctly and it executes well. The params passed to hello-world are incorrect and that will never happen

Reply:
fixed and added test.

Reply:
fixed and added test.